src/pykeen/training/training_loop.py

```diff
@@ -112,11 +113,14 @@ def __init__(
         :param automatic_memory_optimization: bool
             Whether to automatically optimize the sub-batch size during
             training and batch size during evaluation with regards to the hardware at hand.
+        :param collate:
+            Whether to enable collation in the data loader
         """
         self.model = model
         self.optimizer = optimizer
         self.losses_per_epochs = []
         self.automatic_memory_optimization = automatic_memory_optimization
+        self.collate = collate
```

Comment:
@mberr I added a collate flag so we could potentially have both functionalities implemented at the same time

Reply:
I do not think we want to maintain both versions. For benchmarking, just comparing the two branches would also do the trick.

Reply:
Okay fine for me

Reply:
I realize there are several other places where we'd have to write different implementations so this wasn't such a well thought out plan anyway 😅

Reply:
Reverted in 137b8efa